Feature/frw 2013 docs (#1639)
zenml-io/zenml: docs/book/user-guide/starter-guide/connect-to-a-deployed-zenml.md

# Connect to a deployed ZenML

Although the basic functionalities of ZenML work perfectly on your local
machine, you need to connect to **a deployed ZenML server** to use remote
services and infrastructure.

### ZenML Server

When you first get started with ZenML, it is based on the following architecture
on your machine.

![Scenario 1: ZenML default local configuration](../../.gitbook/assets/Scenario1.png)

The SQLite database that you can see in this diagram is used to store
information about pipelines, pipeline runs, stacks, and other configurations. In
the previous pages, we used the `zenml up` command to spin up a local rest
server to serve the dashboard as well. The diagram for this will look as
follows:

![Scenario 2: ZenML with a local REST Server](../../.gitbook/assets/Scenario2.png)

In order to move into production, you will need to deploy this server somewhere
centrally so that the different cloud stack components can read from and write
to the server. Additionally, this also allows all your team members to connect
to it and share stacks and pipelines.

![Scenario 3: Deployed ZenML Server](../../.gitbook/assets/Scenario3.2.png)

### Deployment

> Further down in the Platform Guide you can find a guide on the different 

Reviewer: ```suggestion
Further down in the Platform Guide, you can find a guide on the different 
```

[deployment strategies](../../platform-guide/set-up-your-mlops-platform/deploy-zenml/deploy-zenml.md). 

> This will help you deploy a ZenML Serve ron the backedn of your choice.

Reviewer: ```suggestion
This will help you deploy a ZenML Server on the backend of your choice.
```

Alternatively, if you do not yet want to deploy ZenML yourself, you can use
the [MLOps Sandbox](https://sandbox.zenml.io/) offering to try ZenML with a 

> production stack. You can find a more in-depth guide on this in the next chapter
> on the [Switch to production](switch-to-production.md).

Reviewer: ```suggestion
production stack. You can find a more in-depth guide on this in [the next chapter](switch-to-production.md).
```

```

{% hint style="info" %}

> In Scenario 2, the `zenml up` command implicitly connects the client to the

Reviewer: What do you think of moving this info block below the Scenario 2 figure?

Author: done
